8/sam-phippen-captions.srt

```diff
@@ -700,11 +711,11 @@ thing in here so we can actually just blow this away.
 
 155
 00:12:44,434 --> 00:12:46,537
-That's from private gems.
+That's for private gems.
 
 156
 00:12:47,291 --> 00:12:51,154
-CB: Alright so if you have your own internal Ruby inaud, OK.
+CB: Alright so if you have your own internal Ruby [inaud], OK.
```

Comment:
I can't get this one either. I'd be tempted to remove the `[inaud]` altogether...